sass/MLFacet/_MLFacetValues.scss

```diff
@@ -47,3 +48,27 @@ $active-text-opacity: 0.6;
     }
   }
 }
+
+.coveo-ml-facet-show-more,
+.coveo-ml-facet-show-less {
+  width: 100%;
```

Comment:
I think a lot of those properties/rules could be removed/encapsulated with the mixin `clickable` that you can find in `sass/_Variables.`

Reply:
🤔  Clickable as 2 properties I use here: cursor & text-decoration (for hover). Maybe I can create a similar mixin that I can use for those buttons and the one in the header